docs/understanding-airbyte/typing-deduping.md

```diff
@@ -1,12 +1,10 @@
 # Typing and Deduping
 
-This page refers to new functionality currently available in **early access**. Typing and deduping will become the new default method of transforming datasets within data warehouse and database destinations after they've been replicated. This functionality is going live with [Destinations V2](/release_notes/upgrading_to_destinations_v2/), which is now in early access for BigQuery.
-
-You will eventually be required to upgrade your connections to use the new destination versions. We are building tools for you to copy your connector’s configuration to a new version to make testing new destinations easier. These will be available in the next few weeks.
+This page refers to new functionality added by [Destinations V2](/release_notes/upgrading_to_destinations_v2/). Typing and deduping is the default method of transforming datasets within data warehouse and database destinations after they've been replicated. Please check each destination to learn if Typing and deduping is supported.
 
 ## What is Destinations V2?
 
-At launch, [Airbyte Destinations V2](/release_notes/upgrading_to_destinations_v2) will provide:
+[Airbyte Destinations V2](/release_notes/upgrading_to_destinations_v2) provide:
```

Comment:
```suggestion
[Airbyte Destinations V2](/release_notes/upgrading_to_destinations_v2) provides:
```

Reply:
I *think* it should be provide after a plural, which "destinations" is?  The V2 just confuses me...

Reply:
e.g. "the cat provides scratches" and "the cats provide scratches"?

Reply:
lol the v2 confuses me too 🤷🏼‍♂️